ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -910,10 +912,14 @@ - (void)showViewController:(ORKStepViewController *)viewController goForward:(BO
     
     NSString *progressLabel = nil;
     if ([self shouldDisplayProgressLabel]) {
-        ORKTaskProgress progress = [_task progressOfCurrentStep:viewController.step withResult:[self result]];
-        
-        if (progress.total > 0) {
-            progressLabel = [NSString stringWithFormat:ORKLocalizedString(@"STEP_PROGRESS_FORMAT", nil) ,ORKLocalizedStringFromNumber(@(progress.current + 1)), ORKLocalizedStringFromNumber(@(progress.total))];
+        if (viewController.parentReviewStep && viewController.parentReviewStep.isStandalone) {
+            progressLabel = viewController.parentReviewStep.title;
```

Comment:
I think not. When doing it inside of `willReviewStep`, then either: (1) after the `showViewController: goForward: animated:` call, which would result in a visual glitch (title gets updated after view controller has appeared) or (2) before the `showViewController: goForward: animated:` call, which would instantly override the changed title with the calculated progress.